src/Network/Cookie/RequestCookieJar.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Cake\Network\Cookie;
+
+class RequestCookieJar
+{
+    protected $_cookies = [];
+
+    public function __construct()
+    {
+        foreach ($_COOKIE as $name => $value) {
+            $this->_cookies[$name] = new Cookie($name, $value);
+        }
+    }
+
+    public function get($name) {
+        return $this->_cookies[$name];
```

Comment:
These are cookies sent with a request, so I don't think they should be modified.